web/template/pending-cases.gotmpl

```diff
@@ -24,44 +24,65 @@
+    <div style="text-align:right">
+      <button data-enable-when-selection class="govuk-button govuk-button--secondary" type="submit">Progress worked cases</button>
+    </div>
 
-  <table class="govuk-table">
-    <thead class="govuk-table__head">
-      <tr class="govuk-table__row">
-        <th scope="col" class="govuk-table__header">Donor</th>
-        <th scope="col" class="govuk-table__header">Case</th>
-        <th scope="col" class="govuk-table__header">LPA type</th>
-        <th scope="col" class="govuk-table__header">Received</th>
-        <th scope="col" class="govuk-table__header">Worked</th>
-      </tr>
-    </thead>
-    <tbody class="govuk-table__body">
-      {{ range .Cases }}
+    {{ template "pagination" .Pagination }}
+
+    <hr class="govuk-section-break govuk-section-break--s govuk-section-break--visible govuk-!-margin-top-5">
+
+    <table class="govuk-table">
+      <thead class="govuk-table__head">
         <tr class="govuk-table__row">
-          <th scope="row" class="govuk-table__header">{{ .Donor.DisplayName }}</th>
-          <td class="govuk-table__cell">
-            <a href="{{ sirius (printf "/lpa/person/%d/%d" .Donor.ID .ID) }}" class="govuk-link">
-              {{ .Uid }}
-            </a>
-          </td>
-          <td class="govuk-table__cell">
-            {{ upper .SubType }}
-          </td>
-          <td class="govuk-table__cell">
-            {{ formatDate .ReceiptDate }}
-          </td>
-          <td class="govuk-table__cell">
-          </td>
+          <th scope="col" class="govuk-table__header">Donor</th>
+          <th scope="col" class="govuk-table__header">Case</th>
+          <th scope="col" class="govuk-table__header">LPA type</th>
+          <th scope="col" class="govuk-table__header">Received</th>
+          <th scope="col" class="govuk-table__header">Worked</th>
         </tr>
-      {{ else }}
-        <tr>
-          <td colspan="5">You currently have no cases assigned</td>
-        </tr>
-      {{ end }}
-    </tbody>
-  </table>
-
+      </thead>
+      <tbody class="govuk-table__body">
+        {{ range .Cases }}
+          <tr class="govuk-table__row">
+            <th scope="row" class="govuk-table__header">{{ .Donor.DisplayName }}</th>
+            <td class="govuk-table__cell">
+              <a href="{{ sirius (printf "/lpa/person/%d/%d" .Donor.ID .ID) }}" class="govuk-link">
+                {{ .Uid }}
+              </a>
+            </td>
+            <td class="govuk-table__cell">
+              {{ upper .SubType }}
+            </td>
+            <td class="govuk-table__cell">
+              {{ formatDate .ReceiptDate }}
+            </td>
+            <td class="govuk-table__cell">
+              {{ if .Worked }}
+                <svg class="moj-banner__icon" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" height="25" width="25" viewBox="-5 -5 35 35">
```

Comment:
Does the `moj-banner__icon` add anything here? We'd be better off not using it outside of the `moj-banner` context since it's likely to get side effects.